crates/ironrdp-rdpdr/src/pdu/efs.rs

```diff
@@ -1219,7 +1219,8 @@ where
     }
 
     pub fn decode(header: DeviceIoRequest, src: &mut ReadCursor<'_>) -> PduResult<Self> {
-        ensure_size!(ctx: "DeviceControlRequest", in: src, size: Self::headerless_size());
+        // Ensure the input size includes an additional 20 bytes for padding
+        ensure_size!(ctx: "DeviceControlRequest", in: src, size: Self::headerless_size() + 20);
```

Comment:
suggestion: I think the padding should be part of the "headerless size"

Reply:
Thank you for your suggestion! I changed it to `const`.

Reply:
Thank you!